apps/web/src/styles/7-components/InfoBox/InfoBox.scss

```diff
@@ -0,0 +1,14 @@
+@use "govuk/settings/_colours-applied" as colors;
+
+$pi-govuk-background-colour:  colors.govuk-colour("light-grey");
+$pi-color: rgb(11, 12, 12);
+
+.pi-info-box {
+	background-color: $pi-govuk-background-colour;
+	border-top: solid colors.$govuk-border-colour 1px;
+	border-bottom: solid colors.$govuk-border-colour 1px;
+	color: $pi-color;
```

Comment:
We should use the values from the design system. If its not there, use the closest match.